Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -80,136 +80,161 @@ public function execute()
-                                $array_key = array_search($categoryitem[$categorieskey], $exist_categories_pathname);
+
+                            if (isset($categoriesKey) && ($categoriesKey != '' || $categoriesKey === 0)) {
+                                $array_key = array_search($categoryItem[$categoriesKey], $existingCategoriesPathName);
+
                                 if ($array_key) {
-                                    $alreadyexist[] = $categoryitem[$categorieskey];
-                                } else {
-                                    $strmark = strrpos($categoryitem[$categorieskey], '/');
-                                    $_parentid = '';
-                                    $newcategory = '';
-                                    if ($strmark!=false) {
-                                        $parentpath = substr($categoryitem[$categorieskey], 0, ($strmark));
-                                        $newcategory = substr($categoryitem[$categorieskey], ($strmark)+1);
-                                        $_parentid = array_search($parentpath, $exist_categories_pathname);
-                                    } else {
-                                        $newcategory = $categoryitem[$categorieskey];
-                                        $_parentid = $cat_info->getId();
+                                    $alreadyExist[] = $categoryItem[$categoriesKey];
+                                }
+
+                                else {
+                                    $strMark = strrpos($categoryItem[$categoriesKey], '/');
+                                    $_parentId = '';
+                                    $newCategory = '';
+
+                                    if ($strMark!=false) {
+                                        $parentpath = substr($categoryItem[$categoriesKey], 0, $strMark);
+                                        $newCategory = substr($categoryItem[$categoriesKey], $strMark + 1);
+                                        $_parentId = array_search($parentpath, $existingCategoriesPathName);
                                     }
-                                    if ($_parentid!='' && $newcategory!='') {
-                                        $cateitem = $this->_categoryFactory->create();
-                                        $cateitem->setData($cat_data);
-                                        $parentcategory = $this->_categoryFactory->create();
-                                        $parentcategory->load($_parentid);
-                                        if ($parentcategory->getId()) {
-                                            $cateitem->setParentId($_parentid);
-                                            $cateitem->setPath($parentcategory->getPath());
+                                    else {
+                                        $newCategory = $categoryItem[$categoriesKey];
+                                        $_parentId = $cat_info->getId();
```

Comment:
Please use consistent property naming convention. See the [PSR-1](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md) and  [PSR-2 standards](https://www.php-fig.org/psr/psr-2/#42-properties) regarding property names.